actionpack/test/controller/parameters/mutators_test.rb

```diff
@@ -181,7 +181,7 @@ class ParametersMutatorsTest < ActiveSupport::TestCase
     params = ActionController::Parameters.new(name: "Alex", age: "40", location: "Beijing")
     params.permit!
     params_hash = params.to_h { |key, value| [:"#{key}_modified", value] }
-    assert_equal params_hash.keys, %w(name_modified age_modified location_modified)
+    assert_equal(%w(name_modified age_modified location_modified), params_hash.keys)
```

Comment:
Should I remove parentheses to be consistent with other tests in the same file?

Reply:
I think if there were no parentheses before, we should not add them.  And if there were parentheses before, then we should keep them.

Are these being parentheses from Rubocop's autocorrect?  If so, that is unfortunate. :disappointed:

Reply:
@koic Thank you for the fix in https://github.com/rubocop/rubocop-minitest/pull/259! I tried that branch and it works almost perfectly except for the below corner case.

```
# original
assert_equal @credentials.config, {}

# auto-corrected: syntax error because the {} is interpreted as a block.
assert_equal {}, @credentials.config
```